events/models.py

```diff
@@ -168,6 +188,7 @@ class EventAttendees(models.Model):
     preferences = JSONField(_('Svar'), default=list, blank=True)
     anonymous = models.BooleanField(_('Anonymt'), default=False)
     time_registered = models.DateTimeField(_('Registrerad'))
+    avec_for = models.ForeignKey("self", verbose_name=_('Avec till'), null=True, on_delete=models.SET_NULL)
```

Comment:
To avoid potentially weird errors about this field being required in forms, it would make sense to also add a `blank=True` to this field.

Reply:
Nice, I'll add it.